sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLDriver.scala

```diff
@@ -65,8 +66,15 @@ private[hive] class SparkSQLDriver(val context: SQLContext = SparkSQLEnv.sqlCont
       }
       context.sparkContext.setJobDescription(substitutorCommand)
       val execution = context.sessionState.executePlan(context.sql(command).logicalPlan)
-      hiveResponse = SQLExecution.withNewExecutionId(execution, Some("cli")) {
-        hiveResultString(execution.executedPlan)
+      // Command type sql have been executed when call `context.sql(command).logicalPlan`,
```

Comment:
this comment is not very easy to understand: what is `Command type sql"? maybe:
```
The SQL command has been executed above via `executePlan`, therefore we don't need to wrap it again
with a new execution ID when getting Hive result
```
?

Reply:
Done